README.md

```diff
@@ -6,39 +6,76 @@ The `irb` command from your shell will start the interpreter.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+> **Note**
+>
+> IRB is a default gem of Ruby so you shouldn't need to install it separately.
+>
+> But if you're using Ruby 2.6 or later and want to upgrade/install a specific version of IRB, please follow these steps.
+
+To install it with `bundler`, add this line to your application's Gemfile:
```

Comment:
Since current IRB supports Ruby 2.6 or later, I thought you might as well remove the installation section. I will leave it to you.

Reply:
Sorry I don't get it. I think installation section is still helpful for users with Ruby 2.6+ to upgrade their IRB?
My thinking is, we can encourage power users using IRB as a gem. Because the more people doing that, the faster the feedback loop would be, which would help our development.

Reply:
Oh, sorry. I thought we didn't have to write about how to update default gems because it is the same for default gems, not just IRB. But it's easier to understand if you wrote it. I think you don't need to change it.